src/load/azext_load/data_plane/utils/utils_yaml_config.py

```diff
@@ -97,3 +98,36 @@ def yaml_parse_failure_criteria(data):
                 "requestName"
             ] = name
     return passfail_criteria
+
+
+def parse_regionwise_loadtest_config(regionwise_loadtest_config):
```

Comment:
Should we make it private by renaming to `_parse_regionwise_loadtest_config` ?

Reply:
updated